dev/app.py

```diff
@@ -0,0 +1,28 @@
+import lvgl as lv
+from display import Display
+from widgets import *
+
+NIGHT_THEME = lv.theme_night_init(210, lv.font_roboto_16)
+
+class App():
+
+    def __init__(self, name, display, th=NIGHT_THEME):
+        self.theme = th
+        lv.theme_set_current(self.theme)
+        self.scr = lv.obj()
+        self.name = name
+        self.disp = display
+        self.items = {}
+        self.cont = Container(self.scr)
+
+    def set_title(self, title, font_size=None):
```

Comment:
If the title is set to None will it disappear? Or should there be a remove title function?

Reply:
The idea was that you only use the set_title function if you actually want a title (e.g. the Name app doesn't need a title)

Do you think a remove_title is necessary?

Reply:
That depends, will each game be a separate app from the game menu?

Reply:
I believe so yeah. It's quite a minor change, we can decide a bit further down the line? I just did it like this so that it is not mandatory to have a title for each app